documentation/ipi-install/modules/ipi-install-upstream-adding-a-worker-node-to-an-existing-cluster.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+// Upstream module
+
+[id="adding-a-worker-node-to-an-existing-cluster_{context}"]
+
+= Adding a worker node to an existing cluster
+
+.Procedure
+
+. Retrieve the username and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the username and password. In the following example, the username is `root` and the password is `calvin`.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ echo -ne "root" | base64
+----
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ echo -ne "calvin" | base64
+----
+
+
+. Create a configuration file for the bare metal node.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ vim bmn.yaml
+----
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-<num>-bmc-secret
+type: Opaque
+data:
+  username: <base64-of-uid>
+  password: <base64-of-pwd>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-<num>
+spec:
+  online: true
+  bootMACAddress: <NIC1-mac-address>
+  bmc:
+    address: ipmi://<bmc-ip>
+    credentialsName: openshift-worker-<num>-bmc-secret
+----
++
+Replace `<num>` for the worker number of bare metal node in two `name` fields and `credentialsName` field. Replace `<base64-of-uid>` with the `base64` string of the username. Replace `<base64-of-pwd>` with the `base64` string of the password. Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
+
+[NOTE]
+====
+When using `redfish` or `redfish-virtualmedia`, add the
+appropriate addressing as described in the BMC addressing section. See xref:ipi-install-bmc-addressing_{context}[BMS addressing] for details.
+====
+
+. Create the bare metal node.
++
+[source,bash]
+----
+[kni@openshift-master-0 ~]$ oc -n openshift-machine-api create -f bmn.yaml
```

Comment:
lets change this to from bmn.yaml to bmh.yaml to be mroe aligned with how its called 'baremetal host'

Reply:
everywhere we say openshift-master-0, I think we should label this provisioner. If we decide not to go that route and use master-0 then we need to export the KUBECONFIG to the master host as those commands won't work without that.